addons/project_todo/models/res_users.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+
+from odoo import _, api, fields, models, modules
+
+
+class Users(models.Model):
+    _name = 'res.users'
+    _inherit = ['res.users']
+
+    @api.model
+    def systray_get_activities(self):
+        """ Split To-do and Project activities in systray by removing
+            the single project.task activity represented and doing a
+            new query to split them between private/non-private tasks.
+        """
+        activity_groups = super(Users, self).systray_get_activities()
+        # 1. removing project.task activity group
+        for group in activity_groups:
+            if group.get('model') == 'project.task':
+                activity_groups.remove(group)
+            break
```

Comment:
```suggestion
        for group in activity_groups:
            if group.get('model') == 'project.task':
                activity_groups.remove(group)
                break
```
I guess we should break after removing the group, not after the first iteration

Reply:
> I guess we should break after removing the group, not after the first iteration

Very true! Unfortunately not easy to spot in a test as the project group is almost always the first one :(